app/controllers/concerns/votes.rb

```diff
@@ -0,0 +1,37 @@
+module Votes
+  extend ActiveSupport::Concern
+
+  def votes
+    vote = resource.user_votes.find_by(user: user)
+    if vote
+      vote.vote_type = vote_type
+      vote.save!
+      render json: vote
+    else
+      render json: resource.user_votes.create!(
+        vote_type: vote_type,
+        user: user,
```

Comment:
is is working with extra comma

Reply:
Yes. I have a habit of using trailing commas. I hope its not a bad practice in rails